cfgov/jinja2/v1/_includes/molecules/global-eyebrow.html

```diff
@@ -0,0 +1,49 @@
+{# ==========================================================================
+
+   Global Eyebrow
+
+   ==========================================================================
+
+   Description:
+
+   Creates markup for Global Eyebrow molecule.
+
+   value.is_in_list: Whether the molecule appears in a list or not.
+
+   ========================================================================== #}
+
+<div class="m-global-eyebrow
+            m-global-eyebrow__{{ 'list' if value.is_in_list else 'horizontal' }}">
+    <div class="wrapper">
+        <div class="m-global-eyebrow_tagline">
```

Comment:
So I had it a `p` and it was adding a bottom margin. Instead of overriding that I just made it a `div`. Does that make sense?